README.md

```diff
@@ -231,11 +248,12 @@ _goquery() {
 
 ### Supported Operating Systems
 
-goProbe is currently set up to run on Linux based systems. Tested versions include:
+goProbe is currently set up to run on Linux based systems and Mac OS X. Tested versions include:
```

Comment:
Maybe add: Tested versions include, but are most likely not limited to (since this baby should work with any linux now that it's go-gettable and everything)...?